### ISSUE TITLE
Smoke tests for new/edit product fails on https domain

### ISSUE THREAD

**Issue by grossmannmartin** [shopsys/shopsys#2139]
### To Reproduce

1. Configure application to run in https
2. set first domain to https://localhost:8000 (in domain_urls.yaml)
3. run smoke tests
4. See failed testCreateOrEditProduct `with data set #0 ('admin/product/new/')` and `with data set #1 ('admin/product/edit/1')`

### Expected behavior
Smoke tests pass for https url

### Environment (please complete the following information)
 - OS: MacOs
 - Browser: Does not apply
 - Version 9.1.x-dev after #2117 

### Additional context
In log files is following entry (notice the URL is HTTP, instead of HTTPS as set previously in `domains_urls.yaml`

```
Uncaught PHP Exception Shopsys\FrameworkBundle\Component\Domain\Exception\UnableToResolveDomainException:
"Unable to resolve domain for URL "http://localhost:8000". Check your configuration in "config/domains_urls.yaml"." 
at .../packages/framework/src/Component/Domain/Domain.php line 185
```

Test is in `project-base/tests/App/Smoke/NewProductTest.php` where the client is constructed with first domain URL, but without scheme.

**Comment by grossmannmartin** [comment]
The easiest solution is to set HTTPS in the server config passed to the request. 
```diff
             'HTTP_HOST' => sprintf('%s:%d', parse_url($domainUrl, PHP_URL_HOST), parse_url($domainUrl, PHP_URL_PORT)),
+            'HTTPS' => parse_url($domainUrl, PHP_URL_SCHEME) === 'https',
         ];
```

but then tests start failing on the `Invalid CSRF token` error. 

I figure out it's because in the first client is created session with CSRF tokens prefixed with `https-`
```php
[
    // ...
    "_csrf/https-product_edit_type" => "xxxxxxxxxxxx"
]
```
and with second client when submits the form has data in session without this prefix 
```php
[
    // ...
    "_csrf/product_edit_type" => "xxxxxxxxxxxx"
]
```

when token is validated, it's invalid because of the difference. (to figure out why, take a look at `\Symfony\Component\Security\Csrf\CsrfTokenManager::__construct` method)

If submit of the form is performed on the same client (`$client1`), the test fails on 500 error on attempt to modify the last administrator activity. 
`\Shopsys\FrameworkBundle\Model\Administrator\Activity\AdministratorActivityRepository::getCurrent` do not return the last activity, but throws an exception as none exists (don't know why yet).


_quick'n'dirty solution can be to manually request `tokenId` with the prefix if necessary`
```diff
- $token = $tokenManager->getToken(ProductFormType::CSRF_TOKEN_ID);
+ $token = $tokenManager->getToken(($isSecured ? 'https-' : '') . ProductFormType::CSRF_TOKEN_ID);
```

**Comment by s3tezsky** [comment]
Closed by #2214
